cinder/files/liberty/cinder.conf.controller.Debian

```diff
@@ -87,6 +87,8 @@ control_exchange = cinder
 notification_driver = messagingv2
 {%- endif %}
 
+# Available queries for non-admin user
+query_volume_filters_opt = name,status,metadata,availability_zone,bootable
```

Comment:
this should definitely be configurable, create in map.jinja
'query_volume_filters': []
and then overridable thru model:
control:
  query_volume_filters:
  - name
  - status

make condifition if the query_volume_filters is not [] then print it